index.md

```diff
@@ -11,16 +11,16 @@ Before you get started
 
 ###  What is Rails?
 
-Rails is a framework for building web applications and it uses the Ruby programming language. The important thing that differentiates a web application from a web-site is that an application includes a database on the backend that allows it to store and return data over long periods of time. It's likely that most web pages you visit these days are actually web applications.
+Rails is a framework for building web applications, and it uses the Ruby programming language. The important thing that differentiates a web application from a website is that an application includes a database on the backend that allows it to store and return data over long periods of time. It's likely that most web pages you visit these days are actually web applications.
 
-Rails is also open source means that no one "owns" the Rails framework. While there's a governing body that helps set standards and ensure that it continues to grow and develop with technology, anyone can contribute to it's development. This also means that Rails will always be free to use.
+Rails is also open source, which means that no one "owns" the Rails framework. While there's a governing body that helps set standards and ensures that it continues to grow and develop with technology, anyone can contribute to its development. This also means that Rails will always be free to use.
 
-As you work through the tutorial, don't stress if you don't understand every detail perfectly. This technology is complex and it takes a lot of practice to start feeling like you truely understand how it all works. 
+As you work through the tutorial, don't stress if you don't understand every detail perfectly. This technology is complex, and it takes a lot of practice to start feeling like you truly understand how it all works.
 
 ### Prerequisites
 
-1. Before building anything with Rails you'll need to get your computer set up. Check out our [install instructions]({{ site.baseurl }}/installation.html) to be sure you have everything set up and ready to go.
+1. Before building anything with Rails, you'll need to set up your computer. Check out our [install instructions]({{ site.baseurl }}/installation.html) to be sure you have everything set up and ready to go.
```

Comment:
I like it without the comma because I don't add the pause when I read it. It feels a little awkward to me. However, I can see how the comma could make the sentence more readable.

What if we changed it a little: "Before you can build anything with Rails, you'll need to set up your computer."